compiler_opt/es/blackbox_optimizers.py

```diff
@@ -168,15 +171,17 @@ def set_state(self, state):
 class MCBlackboxOptimizer(BlackboxOptimizer):
   """Class implementing GD optimizer with MC estimation of the gradient."""
 
-  def __init__(self,
-               precision_parameter,
-               est_type,
-               normalize_fvalues,
-               hyperparameters_update_method,
-               extra_params,
-               step_size=None,
-               num_top_directions=0,
-               ga_optimizer=None):
+  def __init__(
+      self,
+      precision_parameter: float,
+      est_type: str,
+      normalize_fvalues: bool,
+      hyperparameters_update_method: str,
+      extra_params: List,
+      step_size: Optional[float] = None,
+      num_top_directions: int = 0,
+      ga_optimizer: Optional[
```

Comment:
this might be a good time to rename this gradient_ascent_optimizer or just 'optimizer'

Reply:
The argument has been renamed to gradient_ascent_optimizer